week_4/quiz-trivia-app/src/app/header/header.component.css

```diff
@@ -0,0 +1,93 @@
+.toggle-switch {
+  position: relative;
+  width: 5.2rem;
+  height: 2.4rem;
+}
+
+.toggle-switch input[type="checkbox"] {
+  display: none;
+}
+
+.toggle-switch label {
```

Comment:
Why do you need the label as absolute?

Reply:
to allow it to overlay the .switch-toggle